src/Bar.php

```diff
@@ -11,4 +11,11 @@ public function foo()
             return true;
         };
     }
+
+    function psr2Violation_bro() {
```

Comment:
- Method name `Bar::psr2Violation_bro` is not in camel caps format
- Visibility must be declared on method `psr2Violation_bro`
- Opening brace should be on a new line